### PR TITLE
uptime formatted.

### PR DESCRIPTION
**Pull request by hthayko**
uptimes are rounded to nearest day/hour/minute/seconds.

### CODE REVIEW

File: src/services/Peers.js [hunk 1]
```diff
@@ -18,10 +19,39 @@ module.exports = {
...
+      var minutes = Math.floor(time / 60);
+      seconds -= minutes * 60;
+      var hours = Math.floor(minutes / 60);
+      minutes -= hours * 60;
+      var days = Math.floor(hours / 24);
+      hours -= days * 24;
+      var dayName = days + ' ' + 'Day' + (days > 1 ? 's' : '');
+      var hourName = hours + ' ' + 'Hour' + (hours > 1 ? 's' : '');
+      var minName = minutes + ' ' + 'min';
+      var secName = seconds + ' ' + 'sec';
```
Comment (wilsonianb) [inline src/services/Peers.js:35]: Can these be consistent with capitalization and pluralization?

Comment (hthayko) [inline src/services/Peers.js:35]: minutes and seconds are long words, so I thought it's a good idea to use
Day(s), Hour(s), min, sec.

File: src/services/Peers.js [hunk 2]
```diff
@@ -18,10 +19,39 @@ module.exports = {
...
+
+      if (days > 0) return { name: dayName, time: days * 60 * 60 * 24 };
+      else if (hours > 0) return { name: hourName, time: hours * 60 * 60 };
+      else if (minutes > 0) return { name: minName, time: minutes * 60 };
+      else return { name: secName, time: seconds };
+    }
+
+    _.map(peers, function(p) {
+      var f = format(p.uptime);
+      p.uptime_truncated = f.time;
```
Comment (wilsonianb) [inline src/services/Peers.js:45]: Why do we need `uptime_truncated`?

File: src/services/Peers.js [hunk 3]
```diff
@@ -18,10 +19,39 @@ module.exports = {
...
+    return peers;
+  },
+
   sortByUptime: function(peers) {
-    peers.sort(function(a, b){
-      var uptime_a = a.uptime ? a.uptime : -1;
-      var uptime_b = b.uptime ? b.uptime : -1;
+    peers.sort(function(a, b) {
+      var uptime_a = a.uptime_truncated || a.uptime || -1;
+      var uptime_b = b.uptime_truncated || b.uptime || -1;
```
Comment (wilsonianb) [inline src/services/Peers.js:54]: I don't think `uptime` still exists

File: src/services/Peers.js (hunk 2 above)
Comment (hthayko) [inline src/services/Peers.js:45]: if we sort by uptime_truncated, than it'll insure no swaps in the region of the same uptime (say there are 20 peers with 25 Days)

Comment (wilsonianb) [inline src/services/Peers.js:45]: Ah, that thing I asked for :)

File: src/services/Peers.js (hunk 3 above)
Comment (hthayko) [inline src/services/Peers.js:54]: what do u mean ?

Comment (wilsonianb) [inline src/services/Peers.js:54]: Nevermind, I mistakenly thought the original `uptime` was lost during `formatUptimes`

**Comment by wilsonianb** [comment]
LGTM :+1:

**Pull request merged**